docs/en/ingest-management/elastic-agent/configuration/providers/docker-provider.asciidoc

```diff
@@ -7,95 +7,46 @@ Provides inventory information from Docker. The available keys are:
 |===
 |Key |Type |Description
 
-|`docker.id`
+|`docker.container.id`
 |`string`
 |ID of the container
 
-|`docker.cmd`
-|`string`
-|Arg path of container
-
-|`docker.name`
+|`docker.container.name`
 |`string`
 |Name of the container
 
-|`docker.image`
+|`docker.container.image.name`
 |`string`
 |Image of the container
 
-|`docker.labels`
+|`docker.container.labels`
 |`string`
 |Labels of the container
-
-|`docker.ports`
-|`string`
-|Ports of the container
-
-|`docker.paths`
-|`object`
-|Object of paths for the container
-
-|`docker.paths.log`
-|`string`
-|Log path of the container
 |===
 // lint enable arg
 
-For example, the Docker provider provides the following inventory:
-
-[source,json]
-----
-[
-    {
-       "id": "1",
-       "mapping:": {"id": "1", "paths": {"log": "/var/log/containers/1.log"}},
-       "processors": {"add_fields": {"container.name": "my-container"}}
-    },
-    {
-        "id": "2",
-        "mapping": {"id": "2", "paths": {"log": "/var/log/containers/2.log"}},
-        "processors": {"add_fields": {"container.name": "other-container"}}
-    }
-]
-----
-
-{agent} automatically prefixes the result with `docker`:
-
-
-[source,json]
----
-[
-    {"docker": {"id": "1", "paths": {"log": "/var/log/containers/1.log"}}},
-    {"docker": {"id": "2", "paths": {"log": "/var/log/containers/2.log"}},
-]
----
-
-To set the log path dynamically in the configuration, use a variable in the
-{agent} policy to return path information from the provider:
+To set the container ID dynamically in the configuration, use a variable in the
+{agent} policy to return container ID information from the provider:
 
 [source,yaml]
 ----
 inputs:
-  - id: "logfile-${docker.paths.log}"
-    type: logfile
-    path: "${docker.paths.log}"
+  - id: 'docker-container-logs-${docker.container.id}'
+    type: filestream
+    paths:
+      - '/var/lib/docker/containers/${docker.container.id}/*-json.log'
 ----
 
-The policy generated by this configuration looks like:
+Sample of the policy generated by this configuration will look like:
 
 [source,yaml]
 ----
 inputs:
-  - id: "logfile-/var/log/containers/1.log"
-    type: logfile
-    path: "/var/log/containers/1.log"
-    processors:
-      - add_fields:
-          container.name: my-container
-  - id: "logfile-/var/log/containers/2.log"
-    type: logfile
-    path: "/var/log/containers/2.log"
-    processors:
-      - add_fields:
-          container.name: other-container
-----
\ No newline at end of file
+  - id: docker-container-logs-b9b898d9c2a1126384d38e9f857b3985480cd05c8e74ffc8b628d92245f5a103
+    paths:
+    - /var/lib/docker/containers/b9b898d9c2a1126384d38e9f857b3985480cd05c8e74ffc8b628d92245f5a103/*-json.log
+  - id: docker-container-596bbd114498253985e6a5c4f0f7bf2d9eb8fcd4fe3e6cb53bdfba0cdc7036c8
```

Comment:
Hmm, how about keeping the `processors` part in place?
```
processors:
      - add_fields:
          container.name: my-container
          container.labels: ....
          ...
```
It would help us understand what metadata will be added.

Reply:
adding those fields - is as well an implementation detail - since it defined [here](https://github.com/elastic/elastic-agent/blob/e7419ddc4e492e7e41acd20ec39f570937affe13/internal/pkg/composable/providers/docker/docker.go#L140-L153), and there is no control over this list

Reply:
The point is that this would show how the events will be enriched with the container's metadata.
Do we have any other resource that would inform the users which metadata are added to the docker container logs?

Reply:
no, we don't, except the keys list. I've added this explanation - https://github.com/elastic/observability-docs/pull/2804/commits/4bdbb67976b5894261439a9f10eabbf8211ee0d2, do you think it is clear enough?

Reply:
Commented there.